app/models/api/v1/report.rb

```diff
@@ -93,6 +93,7 @@ def process_multiple_choice_answer(hash)
     # Make naming more consistent, otherwise we would have crazy sequence of [:answers][:answer][:answer] keys.
     hash[:answer] = hash[:answer].map do |a|
       {
+        id: a[:id],
```

Comment:
I'm not sure this `:id` is the right id here. It is hard to follow but I think the value of `a` for a multiple choice question comes from here: https://github.com/concord-consortium/rigse/blob/master/app/models/saveable/multiple_choice_answer.rb#L11
that value is cached in the report_learner#answers. So if we want to add the id then we need to update multiple_choice_answer and also add a "migration" to clear and rebuild the report_learner cache.  It might be better not do it as a real migration because it will be so dependent on all of the code. Instead it could be a rake task that we have to remember to run on the servers.